src/main/resources/application.yaml

```diff
@@ -43,7 +43,7 @@ info:
 
 authorization:
   case-worker-roles: ${CASE_WORKER_ROLES:caseworker-probate,caseworker-cmc,caseworker-sscs,caseworker-divorce,caseworker-ia}
-  s2s-names-whitelist: ${S2S_NAMES_WHITELIST:sscs,divorce,ccd,em_gw,jui_webapp,pui_webapp,ccd_data,iac}
+  s2s-names-whitelist: ${S2S_NAMES_WHITELIST:sscs,divorce,ccd,em_gw,jui_webapp,pui_webapp,ccd_data,iac,em_stitching_api}
```

Comment:
In the values.yaml this is listed as em_stitching - do we know if it's em_stitching_api or em_stitching?

Reply:
I used the value from     microservice: ${S2S_SERVICE_NAME:em_stitching_api}
in application.yaml in stitching, as that matched what stitching was grabbing from Orch. Let me know if thats incorrect

Reply:
I think it's wrong in values.yaml - can you update it